client/game/menu.py

```diff
@@ -0,0 +1,24 @@
+import pygame
+import pygame_menu
+import main
```

Comment:
We should never import main into any dependency. This will create what's called a circular dependency and limit development going forward. Main needs Menu to initialize at load time and Menu needs Main to initialize at load time. This actually should be breaking the code.

https://en.wikipedia.org/wiki/Circular_dependency

We should follow a path of dependency injection.